.php-cs-fixer.dist.php

```diff
@@ -82,6 +82,7 @@
         'no_superfluous_elseif' => true,
         'no_superfluous_phpdoc_tags' => [
             'allow_mixed' => false,
+            'allow_unused_params' => true,
```

Comment:
Not used in this PR, but is useful when adding a new commented parameter (BC promise).